packages/component/package.json

```diff
@@ -18,10 +18,11 @@
   "homepage": "https://github.com/Microsoft/BotFramework-WebChat/tree/master/packages/component#readme",
   "scripts": {
     "build": "npm run build:typecheck && npm run build:babel-instrumented",
-    "build:babel": "babel --extensions .js,.ts,.tsx --ignore **/*.spec.js,**/*.spec.ts,**/*.spec.tsx,**/*.test.js,**/*.test.ts,**/*.test.tsx,__test__/**/*.js,__test__/**/*.ts,__test__/**/*.tsx --out-dir lib --source-maps inline --verbose src",
+    "build:babel": "babel --extensions .js,.ts,.tsx --ignore **/*.spec.js,**/*.spec.ts,**/*.spec.tsx,**/*.test.js,**/*.test.ts,**/*.test.tsx,__tests__/**/*.js,__tests__/**/*.ts,__tests__/**/*.tsx --out-dir lib --source-maps inline --verbose src",
```

Comment:
Typo long time ago, `__test__` --> `__tests__`.

In Jest, it said we should put it under `__tests__`, https://jestjs.io/docs/en/configuration.html#testregex-string-array-string.

Since the test is run by Jest, it don't need to be transpiled by Babel. Jest will transpile on-the-fly.